### PR TITLE
Demonstrate how to change shape of action buttons

### PR DESCRIPTION
**Pull request by abhishek1508**
The example shows how the default Drop-In UI buttons can be changed from rectangular to circular.

### CODE REVIEW

File: app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+        binding.navigationView.api.routeReplayEnabled(true)
+
+        binding.navigationView.customizeViewStyles {
+            val layoutParams = LinearLayout.LayoutParams(
+                ViewGroup.LayoutParams.WRAP_CONTENT,
+                ViewGroup.LayoutParams.WRAP_CONTENT
+            )
+            audioGuidanceButtonParams = MapboxExtendableButtonParams(
+                R.style.MyCustomAudioGuidanceButtonCircular,
```
Comment (korshaknn) [inline app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt:31]:
```suggestion
                R.style.MapboxStyleAudioGuidanceButton_Circle,
```

File: app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                LinearLayout.LayoutParams(
+                    ViewGroup.LayoutParams.WRAP_CONTENT,
+                    ViewGroup.LayoutParams.WRAP_CONTENT
+                ).apply {
+                    marginStart = 10
+                }
+            )
+            cameraModeButtonParams = MapboxExtendableButtonParams(
+                R.style.MyCustomCameraModeButtonCircular,
```
Comment (korshaknn) [inline app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt:40]:
```suggestion
                R.style.MapboxStyleCameraModeButton_Circle
```

File: app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                LinearLayout.LayoutParams(
+                    ViewGroup.LayoutParams.WRAP_CONTENT,
+                    ViewGroup.LayoutParams.WRAP_CONTENT
+                ).apply {
+                    topMargin = 20
+                    marginStart = 10
+                }
+            )
+            recenterButtonParams = MapboxExtendableButtonParams(
+                R.style.MyCustomRecenterButtonCircular,
```
Comment (korshaknn) [inline app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt:50]:
```suggestion
                R.style.MapboxStyleRecenterButton_Circle,
```

**Review by korshaknn — COMMENTED** [review]
let's use existing styles if possible

File: app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt (hunk 1 above)
Comment (abhishek1508) [reply app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt]: Reused for action buttons. Cannot be done for `Start Navigation`, `Route Preview` and `End Navigation` buttons.

File: app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt (hunk 2 above)
Comment (abhishek1508) [reply app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt]: Reused for action buttons. Cannot be done for `Start Navigation`, `Route Preview` and `End Navigation` buttons.

File: app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt (hunk 3 above)
Comment (abhishek1508) [reply app-preview/src/main/java/com/mapbox/navigation/examples/preview/dropinui/CustomActionButtonsActivity.kt]: Reused for action buttons. Cannot be done for `Start Navigation`, `Route Preview` and `End Navigation` buttons.

**Pull request merged**